server/tests/unit/application.unit.test.js

```diff
@@ -12,6 +13,10 @@ jest.mock('../../src/services/application.services.js', () => ({
     changeStatus: jest.fn(),
 }));
 
+jest.mock('../../src/services/notification.services.js', () => ({
+    sendNotificationApplicationDecision: jest.fn(),
+}));
+
 describe('createApplication', () => {
```

Comment:
I think it also needs the unit tests for the function `sendNotificationApplicationDecision`, because here mocking it in the createApplication is not testing and getting the coverage for the `sendNotificationApplicationDecision`